debian/control

```diff
@@ -0,0 +1,13 @@
+Source: mlxbf-bootctl
+Section: net
+Priority: optional
+Maintainer: Vladimir Sokolovsky <vlad@mellanox.com>
+Build-Depends: debhelper (>= 10)
+Standards-Version: 4.1.2
+
+Package: mlxbf-bootctl
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
```

Comment:
The mlxbf-bootctl userspace tool depends on the kernel module mlx-bootctl. Would it be possible to specify that dependency?

Reply:
Sure, dependency added.